.github/workflows/feature-requests.yml

```diff
@@ -0,0 +1,15 @@
+name: Feature request triage bot
+
+on:
+  schedule:
+    # Run at 13:00 every day
+    - cron: '0 13 * * *'
+
+jobs:
+  feature_triage:
+    if: github.repository == 'angular/angular-cli'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: angular/dev-infra/github-actions/feature-request@354871956801929457d9f08482fb81f7012ed7c8
```

Comment:
Does this require manual version bumps whenever the bot changes? I'm wondering if there's a way to more easily depend on the latest version or at least a release tag?

Reply:
This is automatically updated by renovate.

Reply:
Gotcha, if Renovate handles it, then that should be good enough.

Reply:
It is the process used for keeping the other dev-infra based actions up to date, so we have already proved its success